markdown/dev/guides/writers/notes/en.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Notes
+order: 900
+---
+
+## Do not use tags in the first paragraph!
+
+<Warning>
+Do not use any tags enclosed in `<` and `>` angle brackets in the first paragraph of any documentation file. Due to a limitation on how Open Graph information is created, these tags are interprested and will cause problems.
```

Comment:
You will need to leave an extra line break here to make the parser treat the content of the warning as MDX.

Also, we should probably fix this problem instead of asking users to avoid tags :)

Reply:
Almost forgot: `interprested`is a typo